spec/integration/supported_ssgs_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'swagger_helper'
+
+describe 'SupportedSsgs API' do
+  fixtures :accounts
+
+  path "#{Settings.path_prefix}/#{Settings.app_name}/supported_ssgs" do
+    get 'List all supported SSGs' do
+      tags 'supported_ssg'
+      description 'List all supported SSGs mapped to RHEL minor version'
+      operationId 'ListSupportedSsgs'
+
+      content_types
+      auth_header
+      pagination_params
+      search_params
+
+      include_param
+
+      response '200', 'lists all supported_ssgs requested' do
+        let(:'X-RH-IDENTITY') { encoded_header(accounts(:test)) }
+        let(:include) { '' } # work around buggy rswag
```

Comment:
Not needed for this API

Reply:
It is, unfortunately (`URI::InvalidURIError`) :confused: